core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -226,6 +226,40 @@ oppia.controller('SettingsTab', [
     /********************************************
     * Methods relating to control buttons.
     ********************************************/
+    $scope.previewSummaryTile = function() {
+      alertsService.clearWarnings();
+      $modal.open({
+        templateUrl: 'modals/previewSummaryTile',
+        backdrop: true,
+        controller: [
+          '$scope', '$modalInstance', function($scope, $modalInstance) {
+            $scope.getExplorationTitle = function() {
+              return explorationTitleService.displayed;
+            };
+            $scope.getExplorationObjective = function() {
+              return explorationObjectiveService.displayed;
+            };
+            $scope.getExplorationCategory = function() {
+              return explorationCategoryService.displayed;
+            };
+            $scope.getThumbnailIconUrl = function() {
+              return '/assets/images/subjects/' +
+                explorationCategoryService.displayed + '.svg';
```

Comment:
I've added Lightbulb as the default, is it okay as I've done or would it be better to use the default value in feconf.py (in case it changes in the future).